components/notebook-controller/Dockerfile.konflux

```diff
@@ -46,21 +45,27 @@ RUN microdnf install -y shadow-utils &&\
 ## Create a non-root user with UID 1001
 RUN useradd --uid 1001 --create-home --user-group --system rhods
 
+
+## Set workdir directory to user home
 WORKDIR /home/rhods
 
-COPY --from=builder /workspace/notebook-controller/manager .
-COPY --from=builder /workspace/notebook-controller/third_party/license.txt third_party/license.txt
-# COPY --from=builder /go/pkg/mod/github.com/hashicorp third_party/hashicorp
-ENTRYPOINT ["/manager"]
+## Copy odh-notebook-controller-manager binary from builder stage
+COPY --from=builder /opt/rhods/odh-notebook-controller/bin/manager .
+
+
+## Switch to a non-root user
+USER 1001:0
+
+ENTRYPOINT [ "/manager" ]
 
-LABEL com.redhat.component="odh-kf-notebook-controller-container" \
-      name="managed-open-data-hub/odh-kf-notebook-controller-rhel8" \
+LABEL com.redhat.component="odh-notebook-controller-container" \
```

Comment:
Could you keep the naming of the notebook controllers like:

- notebook-controller -> kf-notebook-controller (Is the notebook controller that comes from kubeflow)
- odh-notebook-controller-> odh-notebook-controller (Is the odh mutated notebook controller)

This is to keep the naming consistency among the project.

Reply:
This is the same label we used in midstream when building with CPaaS. I’d like to know if changing the label would have any other effects?

Reply:
I think we should not change the label, surely not in this PR.

Reply:
Thank you @MohammadiIram for explain this. That's fine for me then